index.html

```diff
@@ -155,6 +155,7 @@
 	xref: ["dom", "accname-1.2", "core-aam-1.2", "infra", "HTML"],
 	definitionMap: []
 	};
+	darkMode: true;
```

Comment:
We removed this config option today in favor of the color-scheme meta tag.
https://lists.w3.org/Archives/Public/spec-prod/2024AprJun/0000.html

Reply:
Thanks @sidvishnoi !